interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,155 @@
                   </leafNode>
                 </children>
               </node>
+              <tagNode name="zone-to-cache">
+                <properties>
+                  <help>Load a zone into the recursor cache</help>
+                  <valueHelp>
+                    <format>txt</format>
+                    <description>The name of the zone to load</description>
+                  </valueHelp>
+                </properties>
+                <children>
+                  <node name="source">
+                    <properties>
+                      <help>Source to load zone</help>
+                    </properties>
+                    <children>
+                      <leafNode name="axfr">
+                        <properties>
+                          <help>Source IP address</help>
+                          <valueHelp>
+                            <format>ipv4</format>
+                            <description>IPv4 address</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="ipv4-address"/>
```

Comment:
Is there a reason not to support IPv6 here? The docs (https://docs.powerdns.com/recursor/lua-config/ztc.html) don't say it's IPv4-only. In fact, it refers to https://docs.powerdns.com/recursor/settings.html#setting-query-local-address and it says that the function it uses (`query-local-address`) supports both IPv4 and IPv6.

Reply:
no reason, fixed